zanata-client-commands/src/main/java/org/zanata/client/commands/push/PushOptionsImpl.java

```diff
@@ -242,4 +244,17 @@ void setValidate(String validate) {
         this.validate = validate;
     }
 
+    @Override
+    public boolean isMyTranslations() {
+        return false;
+    }
```

Comment:
Doesn't this make it impossible to get the real value? This would not print anything, right?

```
foo.setMyTranslations(true);
if (foo.isMyTranslations()) {
    print("It was set and returned successfully");
}
```